### DISCUSSION TITLE
has anyone attempted wayland on nvidia?

### DISCUSSION THREAD

**Discussion by ExpandingMan** [qtile/qtile#3951] [General]
When I try to launch `qtile-wayland` it just crashes... I haven't even tried to debug it, since I don't necessarily expect to be able to get it working, but I'm curious, has anyone tried it?  Does anyone have it working ok?

**Comment by m-col** [comment]
There are a couple of issues that may be related: https://github.com/qtile/qtile/issues/3864, and particularly https://github.com/qtile/qtile/issues/3581. It might be worth reading through and seeing if anything applies to your situation
